examples/telegraf/batch_template.tick

```diff
@@ -0,0 +1,46 @@
+// {alert_name}
```

Comment:
Yes. `generic_batch_example` and `generic_stream_example` could be good. What about `blueprint` or  `model`?